generators/generator-bot-assistant-core/generators/app/templates/dialogs/BotTourDialog/language-generation/en-us/BotTourDialog.en-us.lg

```diff
@@ -0,0 +1,208 @@
+                     "text":"${bodyText}",
+                     "wrap":true,
+                     "size":"medium",
+                     "maxLines":2,
+                     "color":"accent"
+                  }
+               ]
+            }
+         ]
+      }
+   ],
+   "selectAction":{
+      "type":"Action.Submit",
+      "title":"FocusCard",
+      "data":{
+         "intent":"FocusCard"
+      }
+   },
+   "separator": true
+}
+```
+
+# BotTourFocusCardHeader
+- ```
+{
+    "type": "Container",
+    "items": [
+        {
+            "type": "ColumnSet",
+            "columns": [
+                {
+                    "type": "Column",
+                    "width": 10,
+                    "spacing":"none",
+                    "verticalContentAlignment": "center",
+                    "items": [
+                        {
+                            "type": "Image",
+                            "url": "${PlaceholderIconUrl()}",
+                            "horizontalAlignment": "center"
+                        }
+                    ]
+                },
+                {
+                    "type": "Column",
+                    "width":"90",
+                    "spacing":"small",
+                    "verticalContentAlignment": "center",
+                    "items": [
+                        {
+                            "type": "TextBlock",
+                            "text": "${BotTourFocusCardTitle()}",
+                            "size": "large",
+                            "weight": "Bolder"
+                        }
+                    ]
+                }
+            ]
+        }        
+    ]
+}
+```
+
+# BotTourFocusCardBody 
+-```
+{
+    "type": "Container",
+    "items": [
+        {
+            "type": "TextBlock",
+            "text": "${BotTourFocusCardText()}",
+            "wrap": true
+        }        
+    ]
+}
+```
+
+# BotTourFocusCardActions
+- ```
+{
+    "type": "ActionSet",
+    "actions": [
+        {
+            "type": "Action.Submit",
+            "title": "Action A",
+            "data": {
+                "intent": "Action A"
+            }
+        },
+        {
+            "type": "Action.Submit",
+            "title": "Action B",
+            "data": {
+                "intent": "Action B"
+            }
+        },
+        {
+            "type": "Action.Submit",
+            "title": "Action C",
+            "data": {
+                "intent": "Action C"
+            }
+        }
+    ]
+}
+```
+
+# SendActivity_YRvoei_attachment_WGITEE()
+> Bot tour chevron card
+- ```${json(CardTemplate(BotTourHeader(), ExpandableListCardBody(createArray(BotTourBody(BotTourContentTextLong()),BotTourBody(BotTourContentTextShort()),BotTourBody(BotTourContentTextLong()),BotTourBody(BotTourContentTextShort()),BotTourBody(BotTourContentTextLong()),BotTourBody(BotTourContentTextShort()),BotTourBody(BotTourContentTextLong()),BotTourBody(BotTourContentTextShort())), 5), ''))}```
```

Comment:
There's definitely a better expression for this but the real content will not follow the alternating pattern this has.